config.yaml

```diff
@@ -34,6 +34,10 @@ steps:
           target: 'LICENSE'
   - type: generateBuildArtifact
   - type: mirrorProcessedFiles
+  - type: runCommand
+    options:
+      command: "composer normalize"
+      skipConfirmation: true
```

Comment:
```suggestion
  - type: runCommand
    options:
      command: "composer update"
      skipConfirmation: true
  - type: runCommand
       options:
         command: "composer normalize"
         skipConfirmation: true
```

Reply:
Following your suggestion in https://github.com/mteu/basic-project-template/pull/102 I'm updating my PR here.

Reply:
Suggestion directy applied in 507c87f09c784d08b5eb1492fcdc157a95378bf6.